lib/agent/plugins/control-panel/index.js

```diff
@@ -100,10 +110,39 @@ function handle_response(what, err, resp) {
     commands.process(resp.body);
 }
 
+var compatible_with_module_tpm = function (data) {
```

Comment:
maybe this function could be defined in a more global context to be available to other parts of the project

Reply:
done